README.md

```diff
@@ -36,6 +36,7 @@ Python client library to quickly get started with the various [Watson APIs][wdc]
 * You need an [IBM Cloud][ibm-cloud-onboarding] account.
 
 ## Installation
+Note: We are renaming `watson-developer-cloud` to `ibm-watson`
```

Comment:
Should this say `renaming`? Maybe we should say something like we are moving to `ibm-watson` and let them know that all versions prior to `v3.0.0` can still be found in `watson-developer-cloud`?

Reply:
ok sounds good